etc/languages/en-US/help/logs.json

```
@@ -50,5 +50,7 @@
     "MESSAGE_DELETED": "Message Deleted in {{channel}} ({{id}})",
     "BULK_DELETED": "{{amount}} messages were purged in {{channel}} ({{id}}): {{url}}",
     "PURGED": "{{amount}} messages were purged: {{url}}",
-    "MESSAGE_PURGED": "Messages Purged in #{{channel}} ({{id}})"
+    "MESSAGE_PURGED": "Messages Purged in #{{channel}} ({{id}})",
+    "USER_AGE": "**Account Age:** {{age}}",
+    "NEW_ACCOUNT": "**Account Age:** {{age}} (New Account)"
```

I'd prefer not to be duplicating strings if it can be avoided.

```suggestion
    "USER_AGE": "**Account Age:** {{age}}",
    "NEW_ACCOUNT": "(New Account)"
```